bugsnag-android-core/src/main/java/com/bugsnag/android/UserRepository.kt

```diff
@@ -46,4 +40,13 @@ internal class UserRepository(private val prefs: SharedPreferences, private val
         }
         editor.apply()
     }
+
+    fun getDeviceId(): String {
```

Comment:
Add some javadoc comments to explain the intent of this function.

Reply:
Added some KDoc